src/client/components/Maps/Map.js

```diff
@@ -15,6 +14,7 @@ import { getIsMapModalOpen, getSuitableLanguage } from '../../reducers';
 import { setMapFullscreenMode } from './mapActions';
 import './Map.less';
 import mapProvider from '../../helpers/mapProvider';
+import CustomMarker from './CustomMarker';
```

Comment:
import styles at the last line of imports

Reply:
changed